unshipped_data/hit_points/update.py

```diff
@@ -0,0 +1,56 @@
+import csv
+import os
+import yaml
+
+
+def update_yaml(file: str, hit_points: int) -> None:
```

Comment:
I assume you did it like this because `yaml` doesn't support round-trip preservation of the existing file format/comments/etc? I've been meaning to switch us to https://pypi.org/project/ruamel.yaml/ for that reason, but so far I've never actually needed to check one of these in. Makes sense to check this in as-is for now and switch if it ever becomes a problem? (And I expect we don't re-run this often anyway)

Reply:
Exactly, it was to preserve the comments and ordering of the keys so that the diff looked nice.